format/index.md

```diff
@@ -34,20 +34,21 @@ the header `Content-Type: application/vnd.api+json`.
 Clients **MUST** ignore all parameters for the `application/vnd.api+json`
 media type received in the `Content-Type` header of response documents.
 
+Clients that include the JSON API media type in their `Accept` header, **MUST**
+specify the media type there at least once without any media type parameters.
+
 Servers **MUST** send all JSON API data in response documents with
 the header `Content-Type: application/vnd.api+json`.
 
-Servers **MUST** return a `406 Not Acceptable` status code if the
-`application/vnd.api+json` media type is modified by the `ext` parameter in
-the `Accept` header of a request. Otherwise, servers **MUST** return a `415
-Unsupported Media Type` status code if the `application/vnd.api+json` media
-type is modified by the `ext` parameter in the `Content-Type` header of a
-request. Servers **MUST** ignore all other parameters for the
-`application/vnd.api+json` media type in `Accept` and `Content-Type`
-headers.
+Servers **MUST** return a `400 Bad Request` status code if clients violate
+either of the requirements above. The server's response to such violations
+**MUST** contain a JSON API document with an [error object](#error-objects) and
+the `"href"` member of that object **MUST** be
+`"http://jsonapi.org/errors/invalid-media-type-parameter"`.
 
-> Note: These requirements may allow future versions of this specification
-to support an extension mechanism based upon the `ext` media type parameter.
+> Note: These requirements may be loosened in the future to allow subsequent
+versions of this specification to support an extension mechanism in which
+media type parameters are used for extension negotiation.
```

Comment:
Suggestion: "These requirements exist to allow subsequent versions of this specification to use media type parameters for extension negotiation and versioning."